Function fields_for/1 has no local return
Function fields_for/2 only terminates with explicit exception
<<_:40,_:_*8>>
lib/wocky/index/algolia_indexer.ex:16: Function update_object/3 has no local return
lib/wocky/index/algolia_indexer.ex:32: Function do_update_object/2 has no local return
lib/wocky/index/algolia_indexer.ex:37: Function delete_object/2 has no local return
lib/wocky/index/algolia_indexer.ex:38: The pattern {'ok', _} can never match the type
lib/wocky/index/algolia_indexer.ex:42: Function geosearch/3 has no local return
